flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyMessageClientSideSerializationTest.java

```diff
@@ -81,11 +95,47 @@
 		});
 	}
 
-	public NettyMessageSerializationTest(boolean testReadOnlyBuffer, boolean testCompressedBuffer) {
+	public NettyMessageClientSideSerializationTest(boolean testReadOnlyBuffer, boolean testCompressedBuffer) {
 		this.testReadOnlyBuffer = testReadOnlyBuffer;
 		this.testCompressedBuffer = testCompressedBuffer;
 	}
 
+	@Before
+	public void setup() throws IOException, InterruptedException {
+		networkBufferPool = new NetworkBufferPool(10, 1024, 2);
+		BufferPool bufferPool = networkBufferPool.createBufferPool(8, 8);
+
+		inputGate = new SingleInputGateBuilder()
+			.setNumberOfChannels(1)
```

Comment:
this can be removed, default is 1.

Reply:
The `input channel` is changed into another test input channel, thus the problem does not exist now.